qemu/tests/cfg/hv_vapic_test.cfg

```diff
@@ -0,0 +1,41 @@
+- hv_vapic_test:
+    type = hv_vapic_test
+    only Windows
+    start_vm = no
+    cpu_model_flags += hv_crash
+    hv_vapic_flag = hv_vapic
+
+    timeout = 360
+    tmpfs_size = 2G
+    create_tmpfs_cmd = "mount tmpfs {0} -t tmpfs "
+    create_tmpfs_cmd += "&& qemu-img create -f raw {0}/data.raw ${tmpfs_size}"
+    images += " disk1"
+    image_size_disk1 = ${tmpfs_size}
+    image_format_disk1 = raw
+    image_raw_device_disk1 = yes
```

Comment:
Please delete `image_raw_device_disk1 = yes` since `image_raw_device = yes` is for passthrough device or network storage.